### DISCUSSION TITLE
Dashboard widgets not updating anymore

### DISCUSSION THREAD

**Discussion by robinheidrich** [filamentphp/filament#13022] [Help]
### Package

Widgets

### Package Version

v3.2.85

### How can we help you?

The dashboard widgets no longer poll, I'm not sure since which version, but it's been a while. They still do in the demo, so this must have appeared sometime after v3.2.71.

Only the stats overview widget is polled.

Does this happen to anyone else? Or has something changed?

**Comment by leandrocfe** [comment]
Could you provide more info about the issue?

**Reply by robinheidrich** [reply]
Sure. Normally, dashboard widgets would poll without you having to explicitly set it. This works fine for the Stats Overview widget, but the other widgets are not polling. In the demo I can see in the Dev Tools under Network that all widgets are polling. Now, when I look at my site with the latest Filament installed, only the Stats Overview widget is polling.

It used to work, but at some point it stopped working. I would have to try around to find the exact version where it stopped working.

Hope I was able to clear things up!

**Reply by robinheidrich** [reply]
Ok, this is strange, I just installed Filament v3.2.71 and it still only polls the Stats Overview widget. The demo also only polls the chart widgets, not the "Latest Orders" widget.

I could have sworn that the widgets used to poll without using `->poll()`. But I guess that's it.

**Reply by leandrocfe** [reply]
Latest Orders widget is a table widget, right? Did you add `$table->poll(xx)`?

https://filamentphp.com/docs/3.x/tables/advanced/#polling-table-content

**Reply by robinheidrich** [reply]
Yes, I thought table widgets would poll on their own like stats overview or chart widgets. I must have remembered it wrong. Thanks anyway!
